app/assets/javascripts/oxalis/merger_mode.js

```diff
@@ -0,0 +1,276 @@
+// @flow
+import { Modal } from "antd";
+import type { Node, TreeMap } from "oxalis/store";
+import api from "oxalis/api/internal_api";
+
+type NodeWithTreeId = Node & { treeId: number };
+
+type MergerModeState = {
+  treeColors: Object,
+  colorMapping: Object,
+  nodesPerSegment: Object,
+  nodes: Array<NodeWithTreeId>,
+  segementationLayerName: string,
+  nodeSegmentMap: Object,
+  segmentationOpacity: any,
```

Comment:
Flow says that `api.data.getConfiguration("segmentationOpacity")` only returns a boolean type. Any try to avoid this issue (e.g. like conversion to number with `(...): number` did not work out. And I did not want to waist to much time on this so `any`worked out for the flow test.

I know that this is not ideal and that flow should not say that `api.data.getConfiguration("segmentationOpacity")`  only returns a boolean so I wanted to ask you or @daniel-wer about this problem :slightly_smiling_face:

Reply:
Ah, I see. Yes, the typing of `getConfiguration` is very weak. However, it's better to not propagate this. So, please use  `segmentationOpacity: number` and then cast the value to number directly where you call getConfiguration (`(getConfiguration(...)): number`).